core/src/main/java/org/elasticsearch/index/fielddata/ScriptDocValues.java

```diff
@@ -291,4 +291,38 @@ public double geohashDistanceWithDefault(String geohash, double defaultValue) {
             return geohashDistance(geohash);
         }
     }
+
+    class Booleans extends AbstractList<Boolean> implements ScriptDocValues<Boolean> {
+
+        private final SortedNumericDocValues values;
+
+        public Booleans(SortedNumericDocValues values) {
+            this.values = values;
+        }
+
+        @Override
+        public void setNextDocId(int docId) {
+            values.setDocument(docId);
+        }
+
+        @Override
+        public List<Boolean> getValues() {
```

Comment:
actually we could even skip the unmodifiableList wrapper since the wrapped list is already unmodifiable